src/modules/changeIcon.js

```diff
@@ -0,0 +1,14 @@
+// change dot icons btn to type trash btn
+
+export const changeIcon = () => {
```

Comment:
Please remove the export keyword here as you have exported it as default at the end. One block of code should have only a single export by convention.

Reply:
Okay. Thanks